src/spatial/models/PolygonSensorOptimizer.py

```
@@ -209,7 +210,8 @@ def calculate_features_and_scores(self, grid: gpd.GeoDataFrame, polygon: Polygon
             water = self.water_features if not self.water_features.empty else gpd.GeoDataFrame()
 
         except Exception as e:
-            print(f"Warning: Failed to fetch OSM features: {e}")
+#            print(f"Warning: Failed to fetch OSM features: {e}")
+            print(f"Warning: Failed to fetch OSM features: ")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)